scripts/derivechangeset

```diff
@@ -0,0 +1,7 @@
+
+# HootApiDb requires a map layer, though (hence, 'blah').
+step1:
+	hoot derive-changeset $(HOOT_OPTS) -D convert.bounding.box=$(aoi) -D osm.changeset.sql.writer.generate.new.ids=false $(input1) $(input2) $(output) $(jobid) $(DB_URL)/blah
+
```

Comment:
To be able to connect to a hoot api db, hoot requires that you specify a map layer at the end of the URL (doesn't validate it).  In this situation, all I care about is writing my changeset file output to the job status table, so there is no map layer involved.

https://github.com/ngageoint/hootenanny/blob/develop/hoot-core/src/main/cpp/hoot/core/io/HootApiDb.cpp#L824